regression/cbmc/pointer-function-parameters-3/main.c

```diff
@@ -0,0 +1,17 @@
+#include <assert.h>
```

Comment:
Could the tests get better names than "pointer-function-parameters-{3,4,5,6}"?

Reply:
Ok, renamed.